src/client/Client.js

```diff
@@ -227,6 +236,15 @@ class Client extends BaseClient {
     }
   }
 
+  /**
+   * Returns whether the client has logged in, indicative of being able to access
+   * properties such as `user` and `application`.
+   * @returns {boolean}
+   */
+  isReady() {
+    return this.ready;
```

Comment:
Why not just the isReady function? Having ready and isReady doesn't make much sense imo and ready isn't in the typings

Reply:
Ah right, I'll combine those two into just `isReady`.